decisions/0015-env-triad-terms.md

```diff
@@ -0,0 +1,52 @@
+---
+status: proposed / in progress
+date: 2024-08-28
+deciders: Montana Smith, Mark Miller, Sierra Moxen
+consulted: Lee Ann McCue, Natalie Winans
+informed: 
+---
+# Establishing valid terms for MIxS Environmental Triad 
+
+## Context and Problem Statement
+
+Determining what to assert for each of the environmental triad slot (`env_broad_scale`, `env_local_scale`, and `env_medium`) is difficult.
+NMDC will provide a reporducible logic to provide users with a curated lists of valid terms for each slot.
+
+These terms will not be limiting, a user can still use string match to enter terms. This is a temporart allowance until user research and refinment of these queries is completed.
+Any term provided by a user that does **not** come from the provided list will be evaluated.
+
+*This ADR is in progress. Initialy plan has been outlined here & may change*
+
+## Decision Outcome
+
+To provide a reusable method of establishing valid terms, NMDC will provide rules for "general" allowed terms. Meaning the ontology terms that are identified using the "NMDC General Query" could be applied to **any** environmental sample type (MIxS Extensions).
+
+To ensure interoperability and consistancy, the NMDC General Query will be ammended for each environmental sample type.
+- To satisfy NMDC needs, we will start here with soil.
+- This ADR will be updated as environment specific queries are created.
+- No environment specific query should be "cherry picked". Rather any filtering should be accomplished as a general query. (Example, will not remove a specific term by ID, but rather identify what is the rule that can be created to fit a overall need.)
```

Comment:
I'm not too familiar with the subject matter of this ADR (just reviewing it because I was tagged). I found this bullet point confusing and have tried to clarify it here, although I may have deviated from the author's original intention.

```suggestion
- No environment specific query should be "cherry picked"; rather, any filtering should be accomplished as a general query. For example, we will not remove a specific term, but rather identify a rule that accounts for more general needs.
```